server/game/cards/02.4-NMG/Halder.js

```diff
@@ -8,27 +8,19 @@ class Halder extends DrawCard {
                 card.isFaction('thenightswatch') &&
                 (card.getType() === 'attachment' || card.getType() === 'location')
             )),
+            target: {
+                cardCondition: card => card.isFaction('thenightswatch') && card.getType() === 'character'
+            },
             handler: (context) => {
-                this.game.promptForSelect(context.player, {
-                    cardCondition: card => card.isFaction('thenightswatch') && card.getType() === 'character',
-                    activePromptTitle: 'Select character',
-                    source: this,
-                    onSelect: (player, card) => this.onStrCardSelected(player, card, context.kneelingCostCard)
-                });
+                this.game.addMessage('{0} uses {1} to kneels {2} to give {3} +1 STR until the end of the phase', this.controller, this, context.kneelingCostCard, context.target);
```

Comment:
Typo: kneels --> kneel.

Reply:
Opted to change it to the format we've been using elsewhere recently: "Player uses Halder and kneels X to give ..."

Reply:
Even better. :)